src/events/eventToAction.js

```diff
@@ -165,7 +166,7 @@ export default (state: GlobalState, event: Object) => {
       };
     default:
   }
-
+  logWarningToSentry(`Unrecognized event: ${event.type}`);
```

Comment:
Do we need to log warnings to Sentry? I don't think so.

Reply:
Only required here, as we should know which events are not being recognized by our event actions.